pom.xml

```diff
@@ -270,7 +278,7 @@
                 <path>${basedir}/ramls/event.json</path>
                 <path>${basedir}/ramls/instance.json</path>
                 <path>${basedir}/ramls/holdings.json</path>
-                <path>${basedir}/ramls/authority.json</path>
+                <path>${authority.source.path}</path>
```

Comment:
Moving that value as a property also doesn't make sense.

Reply:
it makes sense because authority.source.path is gets a different value for asa-extended profile